check-update-security.sh

```diff
@@ -89,7 +89,7 @@ esac
 # Query ssm to get latest ecs optimized ami
 ami_id=$(aws ssm get-parameters --names $ami_path --region us-west-2 | jq -r '.Parameters[0].Value' | jq -r '.image_id')
 
-user_data=$(mktemp user_data.txt)
+user_data=$(touch user_data.txt)
```

Comment:
mktemp was giving some errors regarding requiring a certain number of 'X's in the file name when testing, so I switched it to making the file normally, since we delete it afterwards anyway. Also, since the environment where github actions is being run is cleaned and shut off after the action is finished, the file need not be deleted later.